Jenkinsfile

```diff
@@ -35,5 +36,20 @@ pipeline {
         }
       }
     }
+    stage('Deploy') {
+      when {
+        anyOf {
+          branch pattern: "master"
+          branch pattern:"feature-*"
+        }
+      }
+      steps {
+        withKubeConfig([credentialsId: 'esa-k8s-token', serverUrl: 'https://1D740396F34543A99F12858947ABAD69.gr7.eu-west-1.eks.amazonaws.com']) {
```

Comment:
Nice but i as a developer would like to have the ability to not only update the image of the app but all k8s objects related to it. So maybe it make send to do `helm upgrade` here

Reply:
kubectl replaced by helm update